.travis.yml

```diff
@@ -10,6 +10,11 @@ env:
   - NODE_ENV=test
   - PINO_EVENT_HUB_NAMESPACE=pinoeventhubns
   - PINO_EVENT_HUB=pinoeventhub
-  - PINO_SHARED_ACCESS_POLICY_NAME=sendPinoEvent 
-  - secure: NWCKKPmwiUJDzeI0tuG9EyOMZ1M8we2yDeG++Qftwk9pYjsOdT2ln6nZEYYMZbZIgMHHhoSfB/F6+ayNWn8fJU7b+WYma2lZjrNqii1szWKrAKKp4BW2Popm/fbl/7btUCrONsOQxA8x9XN8dK3dI0ocdu88eNFd7eRzJrUlOj+DgOto7bJfCnnDCqmAAohR0nD0FzYIRlxvj+w/ze4I60DU3cYdDraOEgw9VzrJx0iTm2cnDgMY3wNFiO5QTcQ6iND73gN7SnG3u/nOgUnTXnhqwX9b4LINe78sSCoKabeLj2gSw66epXQGyJL3VlELGajdBom46GwdJcBS8/iX7h82RZBlsIrNA+jFKEoW7Wehdguv0DKNHlFG+O5qkM++upEMezjTw5nX4f8CLGd4ffrz5JCen5IjBbxafTmZ7ts85tAlrOG9YunxxgySZKEh4mWONdS7jVVKFVEhDXdzfVzHNs4VQ133lrT2JBN21u3QoAlXBshiHdXnlJ5S+nes4r0qQliqKiCtTGOzxD9+hGj9bnhzi0GaUt0W4WAIOd5Y7rh3BMKaDHppIsrHEzRetT38Vf2YxvoRAbZ8Di5qX54SRVJKj2wE6TDdCDdaceWiKsabRlmVd7qWY6mQDFQ2i96JytOad0AAn1RL77PCKaaXpelBKldiXVPtJAj5wqA=
+  - PINO_SHARED_ACCESS_POLICY_NAME=sendPinoEvent
+  - secure: PMCGYxIRYtap+zPJRIOmBlgd4UoVNlEko0nCZFaCgsQoMBCn8FIcDPOVTG8v/di/lvYBtZgr2T+GLcfDIMazzSG2m+nh1LGbErduMollU+jfnHAYmif3vcQaJIAf15EcAXiovfp+tqiinouDyQ+3OjDtLm/NLvBeQAndSo9Ge9eQx+5jXtw17CgfOiTdBmBMqODSF3ZUadQBY5l2YPGIlWjRD5FbAc/tkETiscUCghWBsHUyWg6dw4WswuPr9gjj82a3IBzAy7mIhkeB8aJErqYIpjcclI2VXql9waAxjz+4Ea1S0aY841UlehiStFMDKVByoBbIWYA0/z5wr0feVJSgcMF/qgF975/p+WSOJFpbbyG6NIsV/Ew7llN926Lg5X8w8kkk9l6FC0MJtRsh98QuC+XRhFvt4SuOl9K7LfIXFMmknXclcmALZVzBg6U2gZOkRZFMBttSJrwR3xaU3WFDz4kxtLz3yt9pk26MXyUO2N8lvupoXjk7VNGfDXdy6BriZVPbGBjwia5+l8MIJwEVbt6LZainV/SLZNCjQmd1KRLSSsFtqXPT3ApUkEZnxZSHBQKTm2CG0x25C2Wxcf52bfHpPkyvn31bBc/i3PPs5IIpN5KiLkhnAQOTvIlPfaIzjblC/yLL4OokavUx7I8yf/vzXxI/fzC5HXQt9wE=
```

Comment:
can we set them via the travis UI? It might be easier as they need to be changed often.